src/Sulu/Bundle/ContentBundle/Resources/views/Template/content.html.twig

```diff
@@ -11,6 +11,7 @@
             {% set type = get_type(property.contentTypeName) %}
 
             {% if property.contentTypeName == 'block' %}
+                {% include 'SuluContentBundle:Template:macros/single.html.twig' with {'property': property, 'params': params, 'type': type, 'id': property.name, 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
                 {% include type.template with {'property': property, 'params': params, 'type': type, 'id': property.name, 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
```

Comment:
@berndhep wants this title over the blocks, since it is also fixing https://github.com/sulu-cmf/sulu-standard/issues/324